index.html

```diff
@@ -190,9 +190,30 @@ <h2 class="title">
    
         </section>
 
+        <section class="contact-me">
+            
+        <div class="text-contact">  
+            <h3 id="contact_title">
+                Contact me
+            </h3>
+           
+        </div>
+           <p id="form_text">
+                If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it!
+            </p>
 
-    
- 
+        <form action='https://formspree.io/f/xlekzqzw' method="POST" id="form" >
+            <input type="text" name="name" required placeholder="Name" maxlength="30">
+            <input type="email" name="email" required placeholder="loremipsum@gmail.com">
+            <textarea id="comments" class="input-textarea" name="comment" required placeholder="Write your message here..." maxlength="500"></textarea>
+            <button type="submit">Get in touch</button>
+            
+        </form>
+    </br>
+    </br> 
+        <img src="images/bar.png" alt="A black bar" id="bar">
```

Comment:
- [ ] Good job removing the white space as suggested by the previous reviewer 🥇 However, I noticed a black bar in contact section which was not present before✨ Please make sure to add it according to how it is present in the design guidelines or else remove it because the previous reviewer didn't asked to add it ⛏ See the images below for clarity

Your design before | Your design now | Design guidelines
---| ---| ---|
![image](https://github.com/migcm06/miguel-portfolio/assets/106119105/163db420-5ef8-475d-84a7-fbf886c7c2aa) | ![image](https://github.com/migcm06/miguel-portfolio/assets/106119105/e1dea7f8-5639-466a-adea-e3000583e105) | ![image](https://github.com/migcm06/miguel-portfolio/assets/106119105/bcab24bc-90ba-4fd6-b090-a1a15722f553)